Negotiation/FormatNegotiator.php

```diff
@@ -18,26 +18,15 @@
 use Symfony\Component\HttpFoundation\RequestMatcherInterface;
 use Symfony\Component\HttpFoundation\RequestStack;
 
+/**
+ * @author Ener-Getick <energetick.guigui@gmail.com>
+ */
 class FormatNegotiator extends BaseNegotiator
 {
-    /**
-     * @var array
-     */
     private $map = [];
-
-    /**
-     * @var RequestStack
-     */
     private $requestStack;
```

Comment:
I don't know most of the packages always document the properties...
@lsmith77 @stof what do you think ?